src/components/MultiStepForm/MultiStepForm.tsx

```diff
@@ -122,6 +122,15 @@ const FormContainer: FC<MultiStepFormProps> = ({
 
     setFormErrors({});
 
+    const phoneNumberIndex = formData.findIndex(({ field }) => field.toLowerCase() === 'phone');
+    const phoneNumber = formData[phoneNumberIndex];
+    if (phoneNumber) {
+      const formattedPhoneNumber = formatPhoneNumber(phoneNumber.value as string);
+      if (formattedPhoneNumber) {
+        formData[phoneNumberIndex].value = formattedPhoneNumber;
+      }
+    }
+
```

Comment:
I've added this logic to standardize the format of phone numbers which are sent to our database. With this, all phone numbers will follow the [E.164 international telephone code](https://www.twilio.com/docs/glossary/what-e164). I figured this would be better than some people entering, for example, 07123456789 and others +447123456789.